client/document_test.go

```diff
@@ -124,16 +137,16 @@ func TestSetWithJSON(t *testing.T) {
 	assert.Equal(t, doc.fields["Name"].Type(), LWW_REGISTER)
 	assert.Equal(t, doc.fields["Age"].Name(), "Age")
 	assert.Equal(t, doc.fields["Age"].Type(), LWW_REGISTER)
-	assert.Equal(t, doc.fields["Address"].Name(), "Address")
-	assert.Equal(t, doc.fields["Address"].Type(), OBJECT)
+	// assert.Equal(t, doc.fields["Address"].Name(), "Address")
+	// assert.Equal(t, doc.fields["Address"].Type(), OBJECT)
 
 	//values
 	assert.Equal(t, doc.values[doc.fields["Name"]].Value(), "Alice")
 	assert.Equal(t, doc.values[doc.fields["Name"]].IsDocument(), false)
 	assert.Equal(t, doc.values[doc.fields["Age"]].Value(), int64(27))
 	assert.Equal(t, doc.values[doc.fields["Age"]].IsDocument(), false)
-	assert.Equal(t, doc.values[doc.fields["Address"]].Value(), nil)
-	assert.Equal(t, doc.values[doc.fields["Address"]].IsDocument(), false)
+	// assert.Equal(t, doc.values[doc.fields["Address"]].Value(), nil)
+	// assert.Equal(t, doc.values[doc.fields["Address"]].IsDocument(), false)
 
 	//subdoc fields
```

Comment:
todo: Remove commented code (while at it)

Reply:
✅